src/Grid/DialogDropdown.js

```diff
@@ -0,0 +1,51 @@
+import Dropdown from './Dropdown.js';
+import Input from 'material-ui/Input';
+import React from 'react';
+
+const NumericOperators = [
+  { Value: 'None', Title: 'None' },
+  { Value: 'Equals', Title: 'Equals' },
+  { Value: 'Between', Title: 'Between' },
+  { Value: 'Gte', Title: '>=' },
+  { Value: 'Gt', Title: '>' },
+  { Value: 'Lte', Title: '<=' },
+  { Value: 'Lt', Title: '<' }
+];
+  
+const StringOperators = [
+  { Value: 'None', Title: 'None' },
+  { Value: 'Equals', Title: 'Equals' },
+  { Value: 'NotEquals', Title: 'Not Equals' },
+  { Value: 'Contains', Title: 'Contains' },
+  { Value: 'StartsWith', Title: 'Starts With' },
+  { Value: 'NotStartsWith', Title: 'Not Starts With' },
+  { Value: 'EndsWith', Title: 'Ends With' },
+  { Value: 'NotEndsWith', Title: 'Not Ends With' },
+];
+  
+const BooleanOperators = [
+  { Value: 'None', Title: 'None' },
+  { Value: 'Equals', Title: 'Equals' },
+  { Value: 'NotEquals', Title: 'Not Equals' }
+];
+
+class DialogDropdown extends React.Component {
+  render(){
+    const value = this.props.value === undefined ? 'None' : this.props.value;
+    
+    if (this.props.columnType === 'string') {
```

Comment:
What about a switch for this case?

Reply:
Come on, just one return and set a variable for Operators.